### PR TITLE
Estimation of Pi using Monte Carlo Intergration.

### PR DESCRIPTION
**Pull request by AnandShegde**
## Description
Jaxified and latexified the figures in mc_estimate_pi.ipynb

### Figure Number
book2/10.1

### Figures
|Before PR|After PR|
| :-: |:-:|
|Before_PR_Pi|image
|

## Steps

- [x] Step1 Used Jax instead of Numpy wherever possible.
- [x] Step2 Latexified the images.

## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

**Comment by karm-patel** [comment]
## Figure related comments
1. Keep the number of xicks and yicks the same, also figure does not like squares, you can perhaps use `.set_aspects('equal')` to make the figure square.
2. Get rid of right and top spine using `sns.despine()`

File: notebooks/book2/10/mc_estimate_pi.ipynb [hunk 1]
```diff
@@ -0,0 +1,586 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/10/mc_estimate_pi.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.            plt.figure()
</pre><p>What is use of this line? We can put this line out of conditions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (AnandShegde) [reply notebooks/book2/10/mc_estimate_pi.ipynb:1]: <p>I have fixed the issues that you have mentioned.</p>

File: notebooks/book2/10/mc_estimate_pi.ipynb [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
```
Comment (karm-patel) [inline notebooks/book2/10/mc_estimate_pi.ipynb:1]: <p>ge rid of warnings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (karm-patel) [inline notebooks/book2/10/mc_estimate_pi.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #22.            #         fig,ax = plt.subplots()
</pre><p>Avoid unnecessary commented lines, remove this, check whole code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by karm-patel** [comment]
After resolving these few nits you can create a PR on main repo

**Comment by karm-patel** [comment]
Go ahead for PR

**Pull request merged**